src/scripts/modules/media/nav/nav.coffee

```diff
@@ -57,6 +57,7 @@ define (require) ->
       'click .back-to-top > a': 'backToTop'
       'keydown .searchbar input': 'handleSearchInput'
       'click .searchbar > .clear-search': 'clearSearch'
+      'click .searchbar > .fa-search': 'handleSearch'
```

Comment:
Do other parts of the codebase directly refer to the `fa-` (font-awesome) icon classes? I thought other code relied on an additional class name when targeting them so that we could replace/upgrade the UI (font-awesome) without having to search through references to `fa-` in the JavaScript

Reply:
@philschatz in this class in particular there are a lot of references to `fa` and `fa-` classes, not sure about the rest of the project though.